widget/table_test.go

```diff
@@ -104,7 +104,7 @@ func TestTable_Hovered(t *testing.T) {
 	app.Settings().SetTheme(theme.LightTheme())
 
 	table := NewTable(
-		func() (int, int) { return 3, 5 },
+		func() (int, int) { return 2, 2 },
```

Comment:
What caused this change, was there too many cells for the canvas size, or is there something special about square tables?

Reply:
It is not a change - this was inserted, I think you are seeing an artifact of how github presents the info

Reply:
The additional table (first one) is smaller so we can mouse out of it in the code below

Reply:
Ahh I see